### ISSUE TITLE
[bug]: Copying the markdown of a page doesn't work.

### ISSUE THREAD

**Issue by Cousclou** [makeplane/plane#6128]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

When trying to copy the markdown from the page, you encounter an error in the browser, whether locally or via the domain name, and it's been happening since the upgrade.

```
GET
http://10.94.11.11:32771/fiverr/projects/cc70a71b-f6fa-44eb-a547-47ce3c9c4d6c/pages/083a85d3-b230-4a77-a1a9-f684070b1670/
[HTTP/1.1 200 OK 22ms]

GET
ws://10.94.11.11:32771/live/collaboration?workspaceSlug=fiverr&projectId=cc70a71b-f6fa-44eb-a547-47ce3c9c4d6c&documentType=project_page
[HTTP/1.1 101 Switching Protocols 2ms]

GET
http://10.94.11.11:32771/_next/static/css/2831bc31f562279f.css
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/css/4c5287939a3fe627.css
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/webpack-00d1762104bb55cd.js
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/main-app-786a9f22bb286f06.js

GET
http://10.94.11.11:32771/_next/static/chunks/2364-29069d08dc40ef38.js
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/618f8807-b5255caf8e399037.js
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/2480b5f7-13c7711891114d27.js
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/05882fde-3d1c0fe2792bd75e.js
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/6a1f7758-36f02ec372ccf304.js
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/a76c14b3-81331fcec75a753e.js
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/2f7efde8-35331117ccef8b68.js
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/689cf043-5bfc486db0a80018.js
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/410987da-5df885e785c833ca.js
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/86bc33f4-74b707dfb97e9690.js
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/f5439c6a-2afd0878fd9e3d39.js
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/1e87141b-ffd95bf0eaba6237.js
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/1504-ee28d43f277d5337.js
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/8521-723e811f99f8dc88.js
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/6401-3bf06c13a4b2722f.js
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/72ca3497-9a2bbd104589f0f2.js
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/6807-4d2c6da99355d8dc.js
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/8005-d8d15a853917cf48.js
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/746-fff7edfcdb6bce0e.js
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/7494-d69445ae219d6bc9.js
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/86-cece377db42f162b.js
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/2572-ff3d36c336055da1.js
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/4524-286ed6983696539d.js
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/7955-687f44246bf49f60.js
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/2440-8a6ab98667c8c515.js
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/app/[workspaceSlug]/(projects)/projects/(detail)/[projectId]/pages/(detail)/[pageId]/page-be0d56223586192f.js
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/2937-ad8777fef746a8f0.js
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/1692-789ec321430ce969.js
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/3590-b3d2a3d493f887ac.js
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/8063-ad1e12af91eb4732.js
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/app/[workspaceSlug]/(projects)/projects/(detail)/layout-d7b9e18d406fadea.js
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/417-99654f2ed75c9e17.js
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/app/[workspaceSlug]/(projects)/projects/(detail)/[projectId]/pages/(detail)/layout-f818b0db69ac3bb6.js
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/app/error-ed93c58511db27ee.js
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/app/layout-cd4331dd8ddf72bb.js
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/1790-0681d4f0560f2933.js
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/app/[workspaceSlug]/(projects)/layout-b003b9fcb0986a21.js
[HTTP/1.1 200 OK 0ms]

XHRGET
http://10.94.11.11:32771/api/instances/
[HTTP/1.1 200 OK 13ms]

GET
http://10.94.11.11:32771/_next/static/chunks/app/global-error-5fb08f613a8787c1.js
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/app/not-found-1495ae26baba7da3.js
[HTTP/1.1 200 OK 0ms]

XHRGET
http://10.94.11.11:32771/api/users/me/settings/
[HTTP/1.1 200 OK 57ms]

XHRGET
http://10.94.11.11:32771/api/users/me/profile/
[HTTP/1.1 200 OK 55ms]

XHRGET
http://10.94.11.11:32771/api/users/me/
[HTTP/1.1 200 OK 49ms]

XHRGET
http://10.94.11.11:32771/api/users/me/workspaces/
[HTTP/1.1 200 OK 69ms]

GET
http://10.94.11.11:32771/icons/icon-512x512.png
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/favicon/favicon-16x16.png
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/9518.7b98e71833f3a060.js
[HTTP/1.1 200 OK 0ms]

XHRGET
http://10.94.11.11:32771/api/instances/
[HTTP/1.1 200 OK 5ms]

GET
http://10.94.11.11:32771/_next/static/media/logo-spinner-dark.f21d2bc4.gif
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/5908.8d843a77063ac540.js
[HTTP/1.1 200 OK 0ms]

Intercom not booted or setup incomplete. 5908.8d843a77063ac540.js:1:4062
GET
http://10.94.11.11:32771/_next/static/chunks/3262.ecf25c250bd0576a.js
[HTTP/1.1 200 OK 0ms]

Intercom not booted or setup incomplete. 5908.8d843a77063ac540.js:1:4062
GET
http://10.94.11.11:32771/_next/static/media/logo-spinner-dark.f21d2bc4.gif
[HTTP/1.1 200 OK 0ms]

Intercom not booted or setup incomplete. 5908.8d843a77063ac540.js:1:4062
XHRGET
http://10.94.11.11:32771/api/users/me/
[HTTP/1.1 200 OK 0ms]

XHRGET
http://10.94.11.11:32771/api/users/me/profile/
[HTTP/1.1 200 OK 0ms]

XHRGET
http://10.94.11.11:32771/api/users/me/settings/
[HTTP/1.1 200 OK 0ms]

XHRGET
http://10.94.11.11:32771/api/users/me/workspaces/
[HTTP/1.1 200 OK 0ms]

Intercom not booted or setup incomplete. 5908.8d843a77063ac540.js:1:4062
XHRGET
http://10.94.11.11:32771/api/workspaces/fiverr/workspace-members/me/
[HTTP/1.1 200 OK 69ms]

XHRGET
http://10.94.11.11:32771/api/users/me/workspaces/fiverr/project-roles/
[HTTP/1.1 200 OK 75ms]

XHRGET
http://10.94.11.11:32771/api/workspaces/fiverr/projects/
[HTTP/1.1 200 OK 110ms]

XHRGET
http://10.94.11.11:32771/api/workspaces/fiverr/members/
[HTTP/1.1 200 OK 74ms]

GET
http://10.94.11.11:32771/_next/static/chunks/9402.ed1d7c8c19cfb709.js
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/974.7ed371345ec50639.js
[HTTP/1.1 200 OK 0ms]

GET
http://10.94.11.11:32771/_next/static/chunks/103.566ae061d04443ef.js
[HTTP/1.1 200 OK 0ms]

XHRGET
http://10.94.11.11:32771/api/workspaces/fiverr/user-favorites/?all=true
[HTTP/1.1 200 OK 35ms]

GET
http://10.94.11.11:32771/_next/static/chunks/974.7ed371345ec50639.js
[HTTP/1.1 200 OK 0ms]

XHRGET
http://10.94.11.11:32771/_next/static/media/wa-sqlite.b82ac04f.wasm
[HTTP/1.1 200 OK 0ms]

Error: Failed to initialize database worker: Error: Failed to initialize database: navigator.storage is undefined
    NextJS 11
2364-29069d08dc40ef38.js:1:4063
GET
https://plane.coustech.fr/uploads/60caccc8-c141-4b40-8df8-a8d2b0eee8fe/71f089ae5b1342939a82f413626023e4-Fiverr-Logo.png
[HTTP/1.1 200 OK 0ms]

XHRGET
http://10.94.11.11:32771/api/workspaces/fiverr/users/notifications/unread/
[HTTP/1.1 200 OK 42ms]

XHRGET
http://10.94.11.11:32771/api/instances/changelog/
[HTTP/1.1 200 OK 384ms]

XHRGET
http://10.94.11.11:32771/api/workspaces/fiverr/projects/cc70a71b-f6fa-44eb-a547-47ce3c9c4d6c/pages/083a85d3-b230-4a77-a1a9-f684070b1670/
[HTTP/1.1 200 OK 155ms]

XHRGET
http://10.94.11.11:32771/api/workspaces/fiverr/projects/cc70a71b-f6fa-44eb-a547-47ce3c9c4d6c/
[HTTP/1.1 200 OK 165ms]

XHRGET
http://10.94.11.11:32771/api/workspaces/fiverr/projects/cc70a71b-f6fa-44eb-a547-47ce3c9c4d6c/project-members/me/
[HTTP/1.1 200 OK 84ms]

XHRGET
http://10.94.11.11:32771/api/workspaces/fiverr/projects/cc70a71b-f6fa-44eb-a547-47ce3c9c4d6c/issue-labels/
[HTTP/1.1 200 OK 88ms]

XHRGET
http://10.94.11.11:32771/api/workspaces/fiverr/projects/cc70a71b-f6fa-44eb-a547-47ce3c9c4d6c/members/
[HTTP/1.1 200 OK 100ms]

XHRGET
http://10.94.11.11:32771/api/workspaces/fiverr/projects/cc70a71b-f6fa-44eb-a547-47ce3c9c4d6c/states/
[HTTP/1.1 200 OK 148ms]

XHRGET
http://10.94.11.11:32771/api/workspaces/fiverr/projects/cc70a71b-f6fa-44eb-a547-47ce3c9c4d6c/estimates/
[HTTP/1.1 200 OK 147ms]

XHRGET
http://10.94.11.11:32771/api/workspaces/fiverr/modules/
[HTTP/1.1 200 OK 212ms]

XHRGET
http://10.94.11.11:32771/api/workspaces/fiverr/projects/cc70a71b-f6fa-44eb-a547-47ce3c9c4d6c/cycles/
[HTTP/1.1 200 OK 164ms]

XHRGET
http://10.94.11.11:32771/api/workspaces/fiverr/projects/cc70a71b-f6fa-44eb-a547-47ce3c9c4d6c/views/
[HTTP/1.1 200 OK 210ms]

XHRGET
http://10.94.11.11:32771/fiverr/?_rsc=or5lm
[HTTP/1.1 200 OK 50ms]

XHRGET
http://10.94.11.11:32771/fiverr/projects/?_rsc=or5lm
[HTTP/1.1 200 OK 64ms]

XHRGET
http://10.94.11.11:32771/fiverr/drafts/?_rsc=or5lm
[HTTP/1.1 200 OK 64ms]

XHRGET
http://10.94.11.11:32771/fiverr/notifications/?_rsc=or5lm
[HTTP/1.1 200 OK 57ms]

XHRGET
http://10.94.11.11:32771/fiverr/profile/bc1bd0cb-53bb-416c-9e32-c6114da41d17/?_rsc=or5lm
[HTTP/1.1 200 OK 53ms]

TextSelection endpoint not pointing into a node with inline content (doc) 7494-d69445ae219d6bc9.js:4:465896
GET
ws://10.94.11.11:32771/live/collaboration?workspaceSlug=fiverr&projectId=cc70a71b-f6fa-44eb-a547-47ce3c9c4d6c&documentType=project_page
[HTTP/1.1 101 Switching Protocols 1ms]

XHRGET
http://10.94.11.11:32771/fiverr/active-cycles/?_rsc=or5lm
[HTTP/1.1 200 OK 7ms]

XHRGET
http://10.94.11.11:32771/fiverr/workspace-views/all-issues/?_rsc=or5lm
[HTTP/1.1 200 OK 7ms]

XHRGET
http://10.94.11.11:32771/fiverr/analytics/?_rsc=or5lm
[HTTP/1.1 200 OK 7ms]

XHRGET
http://10.94.11.11:32771/api/workspaces/fiverr/projects/cc70a71b-f6fa-44eb-a547-47ce3c9c4d6c/modules/
[HTTP/1.1 200 OK 88ms]

XHRGET
http://10.94.11.11:32771/fiverr/projects/cc70a71b-f6fa-44eb-a547-47ce3c9c4d6c/pages/?_rsc=or5lm
[HTTP/1.1 200 OK 9ms]

XHRGET
http://10.94.11.11:32771/fiverr/projects/cc70a71b-f6fa-44eb-a547-47ce3c9c4d6c/issues/?_rsc=or5lm
[HTTP/1.1 200 OK 8ms]

GET
http://10.94.11.11:32771/api/assets/v2/workspaces/fiverr/projects/cc70a71b-f6fa-44eb-a547-47ce3c9c4d6c/ebca8230-231c-4b73-9d86-227636901db6/
[HTTP/1.1 302 Found 63ms]

GET
http://10.94.11.11:32771/uploads/60caccc8-c141-4b40-8df8-a8d2b0eee8fe/7510cc3ba69c4265bfeddf0f38fd7490-asZU2.png?response-content-disposition=inline&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=access-key/20241201/us-east-1/s3/aws4_request&X-Amz-Date=20241201T104729Z&X-Amz-Expires=3600&X-Amz-SignedHeaders=host&X-Amz-Signature=331417e429c6651642bdb7c785182a994dd4c865eba98dc78a3be6457d4f6975
[HTTP/1.1 200 OK 4ms]

XHRGET
http://10.94.11.11:32771/fiverr/projects/cc70a71b-f6fa-44eb-a547-47ce3c9c4d6c/pages/083a85d3-b230-4a77-a1a9-f684070b1670/?version=current&_rsc=or5lm
[HTTP/1.1 200 OK 5ms]

GET
http://10.94.11.11:32771/api/assets/v2/workspaces/fiverr/projects/cc70a71b-f6fa-44eb-a547-47ce3c9c4d6c/ebca8230-231c-4b73-9d86-227636901db6/
[HTTP/1.1 200 OK 0ms]

XHRGET
http://10.94.11.11:32771/api/workspaces/fiverr/users/notifications/unread/
[HTTP/1.1 200 OK 60ms]

XHRGET
http://10.94.11.11:32771/api/workspaces/fiverr/projects/cc70a71b-f6fa-44eb-a547-47ce3c9c4d6c/project-members/me/
[HTTP/1.1 200 OK 61ms]

XHRGET
http://10.94.11.11:32771/api/workspaces/fiverr/projects/cc70a71b-f6fa-44eb-a547-47ce3c9c4d6c/
[HTTP/1.1 200 OK 104ms]

La ressource à l’adresse « http://10.94.11.11:32771/api/instances/ » préchargée avec le préchargement de lien n’a pas été utilisée après quelques secondes. Assurez-vous que tous les attributs de la balise de préchargement sont définis correctement. 083a85d3-b230-4a77-a1a9-f684070b1670
La ressource à l’adresse « http://10.94.11.11:32771/api/users/me/ » préchargée avec le préchargement de lien n’a pas été utilisée après quelques secondes. Assurez-vous que tous les attributs de la balise de préchargement sont définis correctement. 083a85d3-b230-4a77-a1a9-f684070b1670
La ressource à l’adresse « http://10.94.11.11:32771/api/users/me/profile/ » préchargée avec le préchargement de lien n’a pas été utilisée après quelques secondes. Assurez-vous que tous les attributs de la balise de préchargement sont définis correctement. 083a85d3-b230-4a77-a1a9-f684070b1670
La ressource à l’adresse « http://10.94.11.11:32771/api/users/me/settings/ » préchargée avec le préchargement de lien n’a pas été utilisée après quelques secondes. Assurez-vous que tous les attributs de la balise de préchargement sont définis correctement. 083a85d3-b230-4a77-a1a9-f684070b1670
La ressource à l’adresse « http://10.94.11.11:32771/api/users/me/workspaces/ » préchargée avec le préchargement de lien n’a pas été utilisée après quelques secondes. Assurez-vous que tous les attributs de la balise de préchargement sont définis correctement. 083a85d3-b230-4a77-a1a9-f684070b1670
XHRGET
http://10.94.11.11:32771/api/workspaces/fiverr/users/notifications/unread/
[HTTP/1.1 200 OK 61ms]

XHRGET
http://10.94.11.11:32771/api/workspaces/fiverr/projects/cc70a71b-f6fa-44eb-a547-47ce3c9c4d6c/
[HTTP/1.1 200 OK 106ms]

XHRGET
http://10.94.11.11:32771/api/workspaces/fiverr/projects/cc70a71b-f6fa-44eb-a547-47ce3c9c4d6c/project-members/me/
[HTTP/1.1 200 OK 66ms]

Uncaught TypeError: e.replace is not a function
    NextJS 42
7494-d69445ae219d6bc9.js:6:65128
Erreur dans les liens source : NetworkError when attempting to fetch resource.
URL de la ressource : moz-extension://d018d347-4fad-48bb-86e6-b4c32264901a/browser-polyfill.js
URL du lien source : browser-polyfill.js.map
XHRGET
http://10.94.11.11:32771/api/workspaces/fiverr/users/notifications/unread/
[HTTP/1.1 200 OK 60ms]

XHRGET
http://10.94.11.11:32771/api/workspaces/fiverr/projects/cc70a71b-f6fa-44eb-a547-47ce3c9c4d6c/project-members/me/
[HTTP/1.1 200 OK 67ms]

XHRGET
http://10.94.11.11:32771/api/workspaces/fiverr/projects/cc70a71b-f6fa-44eb-a547-47ce3c9c4d6c/
[HTTP/1.1 200 OK 105ms]

XHRGET
http://10.94.11.11:32771/api/workspaces/fiverr/users/notifications/unread/
[HTTP/1.1 200 OK 139ms]

XHRGET
http://10.94.11.11:32771/api/workspaces/fiverr/projects/cc70a71b-f6fa-44eb-a547-47ce3c9c4d6c/
[HTTP/1.1 200 OK 188ms]

XHRGET
http://10.94.11.11:32771/api/workspaces/fiverr/projects/cc70a71b-f6fa-44eb-a547-47ce3c9c4d6c/project-members/me/
[HTTP/1.1 200 OK 145ms]


``` 

```
Uncaught TypeError: e.replace is not a function
    NextJS 42
        esc
        serialize
        render
        render
        renderContent
        forEach
        forEach
        renderContent
        serialize
        getMarkdown
        getMarkDown
        action
        onClick
        a_
        aR
        sF
        sF
        sM
        sU
        o4
        iV
        sU
        uR
        uM
        sI
        sR
        sA
        sA
        hydrateRoot
        C
        startTransition
        C
        20893
        n
        n
        20893
        l
        n
        <anonymous>
        O
        c
        <anonymous>
7494-d69445ae219d6bc9.js:6:65128

``` 

### Steps to reproduce

Just copying the markdown of a page.

### Environment

Production

### Browser

Mozilla Firefox

### Variant

Self-hosted

### Version

v0.24.0

**Comment by aaryan610** [comment]
Hey @Cousclou, we have already fixed this bug. It happens for pages with images in them. You can expect it to be fixed with the upcoming release.

**Comment by pushya22** [comment]
HI @Cousclou This issue has been resolved in our latest version 0.24.1. Please upgrade. Feel free to re-open the issue if issue still persists.
